server/rest/src/test/java/org/infinispan/rest/search/BaseRestSearchTest.java

```diff
@@ -270,7 +270,7 @@ private int getCount() throws Exception {
 
    @Test
    public void testMassIndexing() throws Exception {
-      boolean indexEnabled = getConfigBuilder().indexing().enabled();
+      boolean indexEnabled = getConfigBuilder().build().indexing().isEnabled();
```

Comment:
This and other occurrences of this happen because the ```enabled``` getter in IndexingConfigurationBuilder is package local. I'd rather make that public and void the build().